src/app/components/experiment/experiment.component.html

```diff
@@ -31,18 +31,21 @@ <h1 class="t-page-headline">Experiment</h1>
                 <mat-icon>cancel</mat-icon>
             </button>
         </mat-card-title>
-        <mat-card-content>
-            <mat-form-field>
-                <mat-label>Default Template</mat-label>
-                <mat-select (selectionChange)="changeDefaultTemplate($event.value)" [value]="experimentTemplateId">
-                    <mat-option [value]="null">
-                        None
-                    </mat-option>
-                    <mat-option *ngFor="let template of experimentTemplates | keyvalue" [value]="template.key">
-                        {{template.value}}
-                    </mat-option>
-                </mat-select>
-            </mat-form-field>
+        <mat-card-content class="experiment-settings">
+            <div class="default-template-selection t-subheading">
+                <span>
+                    Default Template:
+                </span>
+                <a [routerLink]="['/experiments', experimentId, 'workspace']" [queryParams]="{ template: uiTemplate?.self?.resourceKey?.uiTemplateId ?? 'all-plugins' }">
+                    {{uiTemplate?.name ?? 'All Plugins'}}
+                </a>
+                <button mat-icon-button (click)="showSelectDefaultTemplateDialog()">
+                    <mat-icon>edit</mat-icon>
+                </button>
+                <button mat-icon-button *ngIf="uiTemplate != null" (click)="updateExperimentDefaultTemplate(null)">
+                    <mat-icon>cancel</mat-icon>
```

Comment:
provide a proper accesibility label text for these buttons (cancel is not what this button does)

Reply:
The buttons are now labeled:

![image](https://github.com/UST-QuAntiL/qhana-ui/assets/33042539/efb8a8db-2016-4c8a-a956-ebf9b3683103)